packages/Recognizers.Orchestrator/Microsoft.Bot.Components.Recognizers.Orchestrator.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <PackageId>Microsoft.Bot.Components.Recognizers.Orchestrator</PackageId>
```

Comment:
@clearab should this be a singular Microsoft.Bot.**Component**.Recognizers.Orchestrator

Reply:
Both component and recognizer should be singular, right?

Reply:
Plural it is!